FHEM/lib/signalduino_protocols.hash

```diff
@@ -1,3 +1,47 @@
+################################################################################
+# The file is part of the SIGNALduino project
+#
+# !!! useful hints !!!
+# --------------------
+# name        => ' '       # name of device or group of all devices
+# comment     => ' '       # exact description or example of devices
+# id          => ' '       # number of the protocol definition, each number only once use ((accepted no .)
+# one         => [ , ]     # value pair for a one bit, must be always a positive and negative factor of clockabs (accepted . | example 1.5)
+# zero        => [ , ]     # value pair for a zero bit, must be always a positive and negative factor of clockabs (accepted . | example -1.5)
+# clockabs    => ' '       # value pulse / pause times
+# clockabs    => '-1'      # value pulse / pause times is automatically
+# clockrange  => [ , ]     # min , max of pulse / pause times
+# start       => [ , ]     # value pair for start message
+# sync        => [ , ]     # value pair for sync pulse of signal
+# length_min               # minimum length of bits of the message
+# length_max               # maximum length of bits of the message
+#
+# developId   => 'm'       # the implementation module is under development
+# developId   => 'p'       # protocol is under development or to reserve IDs, the log in the development attribute with developId => 'p' are only used without the other entries
+# developId   => 'y'       # protocol is under development, all logs in the development attribute with developId => 'y' are used
+#
+# preamble    => ' '       # prepend to converted message
+# preamble    => 'u..'     # message is unknown and without module, forwarding SIGNALduino_un or FHEM DOIF
+# preamble    => 'U..'     # message can be unknown and without module, no forwarding SIGNALduino_un but forwarding can FHEM DOIF
+#
+# clientmodule => ' '      # FHEM module for processing
+# filterfunc  => ' '       # SIGNALduino_filterSign | SIGNALduino_compPattern --> SIGNALduino internal filter function, it remove the sign from the pattern, and compress message and pattern
+#                          # SIGNALduino_filterMC --> SIGNALduino internal filter function, it will decode MU data via Manchester encoding
+# float       => [ , ]     # Convert 0F -> 01 (F) to be compatible with CUL
```

Comment:
float gilt nur für MS + MU
Bei MU ist in der sub parse_MU noch eine anpassung notwendig.

Außerdem gibt es zum Senden noch
pause       => [  ]